testing-daml-models/daml-scripts/1_first_script.md

```diff
@@ -0,0 +1,58 @@
+The first thing you need to do when writing a DAML script is to include the DAML script package in
+your dependencies. 
+
+- First, **open the IDE and wait for it to load**
+- After that, click on `daml.yaml`{{open}} to open the `daml.yaml` file.
+- Add the line
+
+  <pre class="file" data-target="clipboard">
+    - daml-script
+  </pre>
```

Comment:
Was too indented when I tried it.

![image](https://user-images.githubusercontent.com/56260209/81645250-8e49e980-9429-11ea-994e-0435d492ffa3.png)

Reply:
fixed.